src/CSET/operators/filters.py

```diff
@@ -21,30 +21,30 @@
 
```

Comment:
```suggestion
from typing import Union
```
Needed for following change.